src/server/pegasus_server_impl.h

```diff
@@ -366,8 +366,15 @@ class pegasus_server_impl : public pegasus_read_service
     void reset_usage_scenario_options(const rocksdb::ColumnFamilyOptions &base_opts,
                                       rocksdb::ColumnFamilyOptions *target_opts);
 
+    void reset_allow_ingest_behind_option(const rocksdb::DBOptions &base_db_opt,
+                                          rocksdb::DBOptions *target_db_opt,
+                                          const std::map<std::string, std::string> &envs);
+
     void reset_rocksdb_options(const rocksdb::ColumnFamilyOptions &base_opts,
-                               rocksdb::ColumnFamilyOptions *target_opts);
+                               rocksdb::ColumnFamilyOptions *target_opts,
```

Comment:
How about renaming base_opts to base_cf_opts, target_opts to target_cf_opts ? To distinguish *_db_opt.